<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarWidget</class>
 <widget class="QWidget" name="CalendarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="summaryGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Summary</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(215, 250, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Distance</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Time</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Pace</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="distLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="timeLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="paceLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../resources/icons.qrc">:/main/icons/runner.xpm</pixmap>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frame">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(233, 255, 198);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBoxWeight">
             <property name="title">
              <string>Weight</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">color: rgb(170, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Max</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="maxWeightLabel">
                <property name="styleSheet">
                 <string notr="true">color: rgb(170, 0, 0);</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="toolTip">
                 <string>Weight</string>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 127);</string>
                </property>
                <property name="text">
                 <string>Average</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="aveWeightLabel">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 127);</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 85, 0);</string>
                </property>
                <property name="text">
                 <string>Min</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="minWeightLabel">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 85, 0);</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBoxBodyfat">
             <property name="title">
              <string>Bodyfat</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_13">
                <property name="styleSheet">
                 <string notr="true">color: rgb(170, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Max</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="bfMax">
                <property name="styleSheet">
                 <string notr="true">color: rgb(170, 0, 0);</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_15">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 127);</string>
                </property>
                <property name="text">
                 <string>Average</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="bfAverage">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 127);</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_17">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 85, 0)</string>
                </property>
                <property name="text">
                 <string>Min</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="bfMin">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 85, 0)</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QGroupBox" name="groupBoxHeartrate">
             <property name="title">
              <string>Heartrate</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="0">
               <widget class="QLabel" name="label_10">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 127);</string>
                </property>
                <property name="text">
                 <string>Average</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="aveHRLabel">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 127);</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="maxHRLabel">
                <property name="styleSheet">
                 <string notr="true">color: rgb(170, 0, 0);</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="styleSheet">
                 <string notr="true">color: rgb(170, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Max</string>
                </property>
               </widget>
              </item>
             </layout>
             <zorder>label_10</zorder>
             <zorder>aveHRLabel</zorder>
             <zorder>maxHRLabel</zorder>
             <zorder>label_8</zorder>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="EditorWidget" name="dayView">
      <attribute name="title">
       <string>Day</string>
      </attribute>
     </widget>
     <widget class="WeekEditor" name="weekView">
      <attribute name="title">
       <string>Week</string>
      </attribute>
     </widget>
     <widget class="MonthEditor" name="monthView">
      <attribute name="title">
       <string>Month</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>All</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="diaryTree">
         <column>
          <property name="text">
           <string>Complete Diary</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>All years</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditorWidget</class>
   <extends>QWidget</extends>
   <header>src/gui/editorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WeekEditor</class>
   <extends>QWidget</extends>
   <header>src/gui/weekeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MonthEditor</class>
   <extends>QWidget</extends>
   <header>src/gui/montheditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
